docs/install/install-aur.md

```diff
@@ -6,6 +6,42 @@ sidebar_position: 4
 
 # Installing Vignette using AUR
 
-something
+If you wish to install Vignette on your Linux computer, you can use any AUR helper of your choice.
+
+The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/).
+
+Here I'll be using `yay`.
```

Comment:
Remove this, and please don't do first-person, this is a professional documentation.